<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GlobeCanTools</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidgetShowData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1071</width>
      <height>531</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>1</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tabWatch">
     <attribute name="title">
      <string>监视</string>
     </attribute>
     <widget class="QGroupBox" name="groupBoxMianInfo">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>650</width>
        <height>500</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>主要参数</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget1">
      <property name="geometry">
       <rect>
        <x>658</x>
        <y>0</y>
        <width>411</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,1">
         <item>
          <widget class="QTabWidget" name="tabWidgetTemperature">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabTemp1">
            <attribute name="title">
             <string>温度1</string>
            </attribute>
            <widget class="QWidget" name="layoutWidget2">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>20</y>
               <width>251</width>
               <height>181</height>
              </rect>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
              <item>
               <layout class="QFormLayout" name="formLayout1">
                <property name="fieldGrowthPolicy">
                 <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
                </property>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_30">
                  <property name="text">
                   <string>温度01：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEditTemp01"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_31">
                  <property name="text">
                   <string>温度02：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEditTemp02"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_32">
                  <property name="text">
                   <string>温度03：</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEditTemp03"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_33">
                  <property name="text">
                   <string>温度04：</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEditTemp04"/>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_34">
                  <property name="text">
                   <string>温度05：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="lineEditTemp05"/>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_35">
                  <property name="text">
                   <string>温度06：</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="lineEditTemp06"/>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_36">
                  <property name="text">
                   <string>温度07：</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="lineEditTemp07"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QFormLayout" name="formLayout2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_37">
                  <property name="text">
                   <string>温度08：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEditTemp08"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_38">
                  <property name="text">
                   <string>温度09：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEditTemp09"/>
                </item>
                <item row="2" column="1">
                 <widget class="QLineEdit" name="lineEditTemp10"/>
                </item>
                <item row="3" column="0">
                 <widget class="QLabel" name="label_40">
                  <property name="text">
                   <string>温度11：</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <widget class="QLabel" name="label_41">
                  <property name="text">
                   <string>温度12：</string>
                  </property>
                 </widget>
                </item>
                <item row="5" column="0">
                 <widget class="QLabel" name="label_42">
                  <property name="text">
                   <string>温度13：</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="1">
                 <widget class="QLineEdit" name="lineEditTemp11"/>
                </item>
                <item row="4" column="1">
                 <widget class="QLineEdit" name="lineEditTemp12"/>
                </item>
                <item row="5" column="1">
                 <widget class="QLineEdit" name="lineEditTemp13"/>
                </item>
                <item row="6" column="0">
                 <widget class="QLabel" name="label_43">
                  <property name="text">
                   <string>温度14：</string>
                  </property>
                 </widget>
                </item>
                <item row="6" column="1">
                 <widget class="QLineEdit" name="lineEditTemp14"/>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_39">
                  <property name="text">
                   <string>温度10：</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
           <widget class="QWidget" name="tabTemp2">
            <attribute name="title">
             <string>温度2</string>
            </attribute>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBoxFaltInfo">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="title">
            <string>错误信息</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxCellVoltage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>电芯电压</string>
         </property>
         <widget class="QWidget" name="layoutWidget3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>381</width>
            <height>231</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Cell01</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell01"/>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Cell02</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell02"/>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Cell03</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell03"/>
             </item>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Cell04</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell04">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Cell05</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell05"/>
             </item>
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Cell06</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell06"/>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Cell07</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell07"/>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Cell08</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell08"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Cell09</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell09">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Cell10</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell10"/>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Cell11</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell11"/>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Cell12</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell12"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Cell13</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell13"/>
             </item>
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Cell14</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell14">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>Cell15</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell15"/>
             </item>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Cell16</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell16"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Cell17</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell17"/>
             </item>
             <item>
              <widget class="QLabel" name="label_18">
               <property name="text">
                <string>Cell18</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell18"/>
             </item>
             <item>
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Cell19</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell19">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Cell20</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEditCell20"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tabData">
     <attribute name="title">
      <string>数据</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1061</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButtonOutputData">
      <property name="geometry">
       <rect>
        <x>950</x>
        <y>470</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>导出数据</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabSet">
     <attribute name="title">
      <string>设置</string>
     </attribute>
     <widget class="QPushButton" name="pushButtonWritePrograme">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>450</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>烧写程序</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBoxSysPra">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>311</width>
        <height>491</height>
       </rect>
      </property>
      <property name="title">
       <string>系统参数</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="labelDevStatus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>0</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Aharoni</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>设备状态</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOpen">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1092</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设备</string>
    </property>
    <addaction name="actionSet"/>
    <addaction name="actionBoardInfo"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSet">
   <property name="text">
    <string>设置</string>
   </property>
  </action>
  <action name="actionBoardInfo">
   <property name="text">
    <string>信息</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
